src/components/ToolMenu/FeatureInfo/index.tsx

```diff
@@ -260,6 +280,13 @@ export const FeatureInfo: React.FC<FeatureInfoProps> = ({
       serializedFeatures[layerName] = new OlFormatGeoJSON().writeFeatures(selectedFeatures);
     });
 
+    const layers: MapLayer[] = Object.keys(features).map(layerName => ({
+      layerName: layerName,
+      index: findMapLayerIndex(layerName)
+    })).sort((a, b) => b.index - a.index);
+
+    setActiveTabKey(layers[0].layerName);
```

Comment:
What happens if `layers[0]` is not available?

Reply:
A check is added here to test if layers is present and has at least one value.